cs-couchbase/src/test/java/io.cloudslang.content.couchbase/execute/ExecutorTest.java

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ * (c) Copyright 2017 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.content.couchbase.execute;
+
+import io.cloudslang.content.couchbase.entities.inputs.BucketInputs;
+import io.cloudslang.content.couchbase.entities.inputs.CommonInputs;
+import io.cloudslang.content.httpclient.CSHttpClient;
+import io.cloudslang.content.httpclient.HttpClientInputs;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.net.MalformedURLException;
+import java.util.HashMap;
+
+import static io.cloudslang.content.couchbase.utils.InputsUtil.getHttpClientInputs;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.whenNew;
+
+/**
+ * Created by TusaM
+ * 4/11/2017.
+ */
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({CSHttpClient.class, Executor.class})
+public class ExecutorTest {
+    @Mock
+    private CSHttpClient csHttpClientMock;
+
+    private Executor toTest;
+    private HttpClientInputs httpClientInputs;
+
+    @Before
+    public void init() throws Exception {
+        whenNew(CSHttpClient.class).withNoArguments().thenReturn(csHttpClientMock);
+        when(csHttpClientMock.execute(any(HttpClientInputs.class))).thenReturn(new HashMap<String, String>());
+        toTest = new Executor();
+        httpClientInputs = getHttpClientInputs("someUser", "credentials", "", "",
+                "", "", "GET");
+    }
+
+    @After
+    public void tearDown() {
+        toTest = null;
```

Comment:
the tearDown of the attributes is not really necessary when in the @Before you create new instances of the attributes ( toTest, httpClientInputs), Junit4.x and Java's garbage collector will take care of them ... but you can leave them be, it's a thing of  taste :)